provider-gui/services/contract/contractOverview.ts

```diff
@@ -0,0 +1,98 @@
+import { ApiPromise, Keyring, WsProvider } from '@polkadot/api'
+import { GovernanceStatus } from '@prosopo/captcha-contract'
+import { ProcaptchaClientConfigOutput } from '@prosopo/types'
+import { ProsopoCaptchaContract } from '@prosopo/contract'
+import { ContractAbi as abiJson } from '@prosopo/captcha-contract'
+import { hexToString } from '@polkadot/util'
+
+const deployedContracts = {
+    rococo: {
+        wsProviderUrl: 'wss://rococo-contracts-rpc.polkadot.io:443',
+        contracts: ['5HiVWQhJrysNcFNEWf2crArKht16zrhro3FcekVWocyQjx5u'],
```

Comment:
is there any ability to enter our own contract here? I'm imaging this tool being hosted somewhere and I've just deployed a contract, don't want to recompile and restart the provider status tool so would be easier just to type in the contract address manually. Don't have to add now tho, could just put it into a ticket

Reply:
https://github.com/prosopo/captcha/issues/771